src/Initializer/MemoryAllocator.cpp

```diff
@@ -79,7 +79,7 @@
 
 void* seissol::memory::allocate(size_t i_size, size_t i_alignment, enum Memkind i_memkind)
 {
-    void* l_ptrBuffer;
+    void* l_ptrBuffer{};
```

Comment:
Maybe `=nullptr` would be more verbose?

Reply:
Imo that is clear enough, nulptr is obviously the default value